### PR TITLE
samples: openthread: bug fixes

### PR DESCRIPTION
**Pull request by lmaciejonczyk**
It fixes https://projecttools.nordicsemi.no/jira/browse/KRKNWK-6261
Depends on https://github.com/nrfconnect/sdk-zephyr/pull/316

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: fde8b6badf955c98f535b1c51903d682b45173f7
+      revision: pull/316/head
```
Comment (MarekPorwisz) [inline west.yml:49]: To be updated.

**Comment by rlubos** [comment]
Please rebase, we need a green CI on the PR.

**Comment by lmaciejonczyk** [comment]
> Please rebase, we need a green CI on the PR.

https://github.com/nrfconnect/sdk-zephyr/pull/316 need to be merged anyway and after west.yml will be updated

**Comment by rlubos** [comment]
> nrfconnect/sdk-zephyr#316 need to be merged anyway and after west.yml will be updated

Yes, but we cannot merge it without verifying with the CI that the `sdk-zephyr` PR does not break something in `sdk-nrf`. That's the process we have, not my fussiness.

**Comment by lmaciejonczyk** [comment]
> > [nrfconnect/sdk-zephyr#316](https://github.com/nrfconnect/sdk-zephyr/pull/316) need to be merged anyway and after west.yml will be updated
> 
> Yes, but we cannot merge it without verifying with the CI that the `sdk-zephyr` PR does not break something in `sdk-nrf`. That's the process we have, not my fussiness.

Sorry about that. I understand it now. The rebase is done.

**Comment by rlubos** [comment]
https://github.com/nrfconnect/sdk-zephyr/pull/316 was merged, please update `west.yml`: ad0444b058ef5b3586784425c5a353cd5fe9a315

**Pull request merged**